src/vs/workbench/contrib/scm/browser/activity.ts

```diff
@@ -116,10 +116,12 @@ export class SCMActiveRepositoryController extends Disposable implements IWorkbe
 		this._activeRepositoryNameContextKey = ActiveRepositoryContextKeys.ActiveRepositoryName.bindTo(this.contextKeyService);
 		this._activeRepositoryBranchNameContextKey = ActiveRepositoryContextKeys.ActiveRepositoryBranchName.bindTo(this.contextKeyService);
 
-		this.titleService.registerVariables([
+		const titleVariables = [
 			{ name: 'activeRepositoryName', contextKey: ActiveRepositoryContextKeys.ActiveRepositoryName.key },
 			{ name: 'activeRepositoryBranchName', contextKey: ActiveRepositoryContextKeys.ActiveRepositoryBranchName.key, }
-		]);
+		];
+		this.titleService.registerVariables(titleVariables);
```

Comment:
It would be cool if `registerVariables` would return a disposable to unregister the variables, however, with the current layering of the title bar part (multi window aware...) it's not that simple so out of scope for this PR.